### PR TITLE
PAYARA-3225-Location-Header-Not-Present-with-WebApplicationException - Payara4

### PR DESCRIPTION
**Pull request by mulderbaba**

### CODE REVIEW

**Comment by mulderbaba** [comment]
Jenkins, quick test Payara 4 please

**Comment by MeroRai** [comment]
jenkins test please

File: appserver/payara-appserver-modules/microprofile/opentracing/src/main/java/fish/payara/microprofile/opentracing/jaxrs/JaxrsContainerRequestTracingExceptionMapper.java [hunk 1]
```diff
@@ -44,31 +44,38 @@
...
         if (exception instanceof WebApplicationException) {
-            status = ((WebApplicationException) exception).getResponse().getStatusInfo();
-        }    
-        
+            Response response = ((WebApplicationException) exception).getResponse();
+            location = response.getLocation();
+            status = response.getStatusInfo();
+        }
+
+        Response.ResponseBuilder responseBuilder = Response.status(status);
```
Comment (MattGill98) [inline appserver/payara-appserver-modules/microprofile/opentracing/src/main/java/fish/payara/microprofile/opentracing/jaxrs/JaxrsContainerRequestTracingExceptionMapper.java]: Why is a new response being created here at all? Shouldn't the WebApplicationException response be used as a base if it exists?

Comment (Pandrex247) [reply appserver/payara-appserver-modules/microprofile/opentracing/src/main/java/fish/payara/microprofile/opentracing/jaxrs/JaxrsContainerRequestTracingExceptionMapper.java]: Already merged into master, raised issue PAYARA-3262 to track.

**Pull request merged**